spec/features/category_spec.rb

```diff
@@ -36,4 +41,10 @@
     find_link("Destroy").click
     expect(page).to have_content "Category was successfully destroyed."
   end
+
+  scenario 'should show all carriers' do 
+    visit "/categories/#{category.id}"
```

Comment:
You can use a URL helper here.
```suggestion
    visit category_path(category)
```

Reply:
I wanted to follow the pattern used.

Reply:
:+1: